src/controllers/UserController.js

```diff
@@ -37,6 +37,33 @@ export default class UserController {
     }
   }
 
+  /**
+ * @name getUser
+ * @description Allows user to sign in user
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @returns {object} The API response
+ */
+  static async getUser(req, res) {
```

Comment:
**PLEASE CHANGE THE NAME OF THIS METHOD TO `login` or something similar that would describe what this method is achieving**

Reply:
name changed to login.